### ISSUE TITLE
Training data

### ISSUE THREAD

**Issue by hokmund** [AhmetHamzaEmra/Intelegent_Lock#1]
Could you please provide datasets which were used for the training?

**Comment by AhmetHamzaEmra** [comment]
Training set contains a lot of photos of my friends. I am sorry but first I need to get their permission.

**Comment by hokmund** [comment]
Could you then provide some metrics (e.g., accuracy or EER) for your liveness detection model?
It will allow others to compare their models to yours.
Thank you in advance

**Comment by sandys** [comment]
can you mention name of datasets we can use to train ? we have access to many datasets, but dont know which one to use here

**Comment by AhmetHamzaEmra** [comment]
I prepare the dataset so it is not from online sources. but since it contains my friends photos I can not put dataset online.

**Comment by UrstrulyRoot** [comment]
could you tell me the kind of data that needed to finetune it?
can i take pictures of my friends as a live image and pictures of the pictures as img-not-live and train it?

**Comment by motoleisure** [comment]
@AhmetHamzaEmra what kind of data do I need? How to tell the live and not-live just using photos ? Thanks !!

**Comment by AhmetHamzaEmra** [comment]
Any photos that don't contain any electronic device or printed form of pictures of a face are live pictures. and for the not-live dataset use the opposite.
